src/ui/components/Share/component.ts

```diff
@@ -0,0 +1,34 @@
+import Component, { tracked } from '@glimmer/component';
+
+export default class extends Component {
+  @tracked fullUrl = '';
+  @tracked tinyUrl = '';
+  @tracked fullUrlCopied = false;
+  @tracked tinyUrlCopied = false;
+
+  constructor(options) {
+    super(options);
+    this.fullUrl = window.location.href;
+  }
+
+  generateTinyurl() {
+    let url = "https://tinyurl.com/api-create.php?url=" + this.fullUrl;
+    this.tinyUrl = 'https://tinyurl.com/blablabla';
```

Comment:
I believe this should return url.

Reply:
Sorry, this is just a placeholder for the moment.